### DISCUSSION TITLE
would/should it be able to be ported to fabric

### DISCUSSION THREAD

**Discussion by pat600130** [Sinytra/Connector#1601] [Ideas]
so my idea is to make this but for fabric so i can run forge mods on fabric but idk how doable it is.

**Comment by VinnyStalck** [comment]
The Kilt project is trying to do just that
https://github.com/KiltMC

**Comment by WitherTM** [comment]
![image](https://github.com/user-attachments/assets/61014718-7cc7-472d-8b0e-a9f946b8572c)

**Reply by ert11er** [reply]
man this meme got more upvoted than the recommendation
